ui/app/serializers/config-ui/message.js

```diff
@@ -35,12 +32,8 @@ export default class MessageSerializer extends ApplicationSerializer {
   serialize() {
     const json = super.serialize(...arguments);
     json.message = encodeString(json.message);
-    json.link = {
-      title: json?.link_title || '',
-      href: json?.link_href || '',
-    };
-    delete json?.link_title;
-    delete json?.link_href;
+    const [linkTitle] = Object.keys(json.link);
+    json.link = isEmpty(linkTitle) ? null : json.link;
```

Comment:
Adding these checks since the json.link could be set like `{"": ""}`. If a user edits the link form field, the fields save the object with a key and value as empty strings.

Reply:
This seems like something the component shouldn't allow 🤔 Likely outside the scope of this PR. Or does it seem like the object transform should handle this case? There are other parts of the app where this has been an issue, I think. 

The `serialize` method in transforms/object.js could be updated so if the object is just empty strings, it returns`{}` 
```js
 serialize(value) {
    if (typeOf(value) !== 'object') {
      return {};
    } else {
      return JSON.stringify(value) === '{"":""}' ? {} : value
    }
  }
```
Also an empty string is falsy, so there's no need to import `isEmpty`, the check could be 
```js
!linkTitle ? null : json.link;

Reply:
Great idea to add to the transform :D